src/oop/oop1.py

```diff
@@ -17,3 +17,39 @@
 #     pass
 #
 # Put a comment noting which class is the base class
+
+
+#base class
+class Vehicle:
+    def __init__(self):
+        pass
+
+class FlightVehicle(Vehicle):
+    def __init__(self):
+        super().__init__()
+        pass
+
+class GroundVehicle(Vehicle):
+    def __init__(self):
+        super().__init__()
+        pass
+
+class Starship(FlightVehicle):
+    def __init__(self):
+        super().__init__()
+        pass
+
+class Airplane(FlightVehicle):
+    def __init__(self):
+        super().__init__()
+        pass
+
+class Car(GroundVehicle):
+    def __init__(self):
+        super().__init__()
+        pass
+
+class Motorcycle(GroundVehicle):
+    def __init__(self):
+        super().__init__()
+        pass
```

Comment:
Well done here. As we spoke about we could have simply put `pass` inside the bodies of the classes here.